tensorflow-framework/src/main/java/org/tensorflow/framework/activations/ELU.java

```diff
@@ -72,16 +76,96 @@ public ELU(double alpha) {
     this.alpha = alpha;
   }
 
-  /** {@inheritDoc} */
-  @Override
-  public Operand<T> call(Ops tf, Operand<T> input) {
+  /**
+   * Creates a new ELU from a configuration Map
+   *
+   * @param config the configuration map, if the map contains an entry for {@code alpha} that value
+   *     is used, otherwise {@link #ALPHA_DEFAULT} is used.
+   */
+  public ELU(Map<String, Object> config) {
```

Comment:
What happens if you supply the configuration for e.g. `ReLU` to this constructor? I would expect that to be an error, but I think in this case it would silently ignore it and default alpha. That doesn't seem like ideal behaviour.

Reply:
Or maybe change the scope of this constructor to be protected or package-private?